src/AdornedRulerPanel.cpp

```diff
@@ -1417,7 +1417,7 @@ void AdornedRulerPanel::ReCreateButtons()
       this,
       bmpRecoloredUpSmall, bmpRecoloredDownSmall, 
       bmpRecoloredUpHiliteSmall, bmpRecoloredHiliteSmall, 
-      bmpPlayPointer, bmpPlayPointer, bmpPlayPointer,
+      bmpCogwheel, bmpCogwheel, bmpCogwheel,
```

Comment:
Now, I have reviewed to the end, and I understand the button is an unchanging gearwheel.

OK, I have no objections to anything.